pulsar-broker/src/test/java/org/apache/pulsar/client/api/BrokerServiceLookupTest.java

```diff
@@ -731,6 +731,9 @@ public void testModularLoadManagerSplitBundle() throws Exception {
             // broker-2 loadManager is a leader and let it refresh load-report from all the brokers
             pulsar.getLeaderElectionService().close();
 
+            Awaitility.await().atMost(5, TimeUnit.SECONDS)
```

Comment:
We can even use the default timeout here 
```suggestion
            Awaitility.await()
```